# ViewpointArray.msg - Custom message for ViewpointArray structure

Viewpoint[] viewpoints